glow.py

```diff
@@ -293,7 +293,7 @@ def infer(self, spect, sigma=1.0):
 
     @staticmethod
     def remove_weightnorm(model):
-        waveglow = copy.deepcopy(model)
+        waveglow = model
```

Comment:
let's maintain the deepcopy

Reply:
The deepcopy fails if the model has been automatically converted previously. If automatic conversion is to be removed we can reinstate deepcopy